ui/app/src/models/Version.ts

```diff
@@ -14,7 +14,7 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { LegacyItem, SandboxItem } from './Item';
+import { DetailedItem, LegacyItem } from './Item';
```

Comment:
Please check if this can work with a BaseItem so that it could receive Sandbox or Detailed item alike.